pkg/features/kube_features.go

```diff
@@ -1000,7 +1000,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CSIVolumeHealth: {Default: false, PreRelease: featuregate.Alpha},
 
-	SkipReadOnlyValidationGCE: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
update the doc on the gate above and clarify the description of what it does?

Reply:
ACK